pkg/client/interface.go

```diff
@@ -18,8 +31,21 @@ import (
 	"pocket/pkg/observable"
 )
 
+// TxClient provides an interface for initiating transactions on a blockchain.
```

Comment:
it says `on a blockchain`, but do we specifically mean on the `pocket blockchain`?

I'm asking because we use tendermint/comet/cosmos types (because that's how we're building OUR blockchain) but the underlying usage is meant to be blockchain agnostic (e.g. etheruem, solana, etc...). This sort of comment makes it a bit misleading. 

Leaving in one place but applicable everywhere